packages/sd-jwt/src/SdJwtOptions.ts

```diff
@@ -0,0 +1,28 @@
+import type { HashName, Key } from '@aries-framework/core'
+import type { DisclosureFrame } from 'jwt-sd'
+
+export type SdJwtCreateOptions<Payload extends Record<string, unknown> = Record<string, unknown>> = {
```

Comment:
@TimoGlastra this is the input now for the creation of a sd-jwt-vc. Holder binding is required and the issuer has to provide a did, for the `iss` field and a key. 

Very curious to see if there are any alternatives for this as I think the options are a bit of a mess...

Reply:
Why does the issuer need to provide both an issuerDid and an issuerKey? I think with other places we've required to pass a key reference within the did then (`did:key:123#123`) and based on that we can get the issuerDid and issuerKey.

Usually I try to avoid two params that can clash (as you can provide a key that doesn't belong to the did). We can call the parameter the `keyId` (or something like that, forgot what it's called) and also add it to the header of the JWT